src/components/CompetitionPage.vue

```diff
@@ -1,291 +1,413 @@
-          and cooperation, Space Apps 2021 will have winners. In previous years,
-          awards have been given for the following categories. However, NASA
-          reserves the right to change the award categories. <br /><br />
-          <ul class="circle-bullet">
-            <li>Best Use of Data</li>
-            <li>Best Use of Hardware or Technology</li>
-            <li>Galactic Impact</li>
-            <li>Most Inspirational</li>
-            <li>Best Mission Concept</li>
-            <li>Best Use of Science</li>
-          </ul>
+              <li class="guideline-item">
+                <h4 class="guideline-title">Join the Chat Rooms</h4>
+                <p>
+                  Chat rooms will open more than a week before the hackathon.
+                  Connect with the global community, find teammates, and join
+                  challenge-specific channels. Subject-Matter Experts and Space
+                  Apps Ambassadors will be available to answer your questions.
+                </p>
+              </li>
 
-          <br />
-          Space Apps winners will be featured on the Space Apps website and
-          social media, and they will receive an invitation to present their
-          projects to NASA personnel (and space agency partners, if applicable).
-          <br/> <br/>
-          If travel is deemed safe, the winners will also be invited to visit a
-          NASA site with the Space Apps Global Organizing Team to view a
-          spacecraft launch. Each member of each winning team and one guest each
-          will be invited and will be responsible for covering their own costs
-          for their personal travel, meals and incidental expenses, and
-          accommodation expenses. Winners typically conduct fundraising
-          activities to cover the costs of their trips.
+              <li class="guideline-item">
+                <h4 class="guideline-title">Join a Team</h4>
+                <p>
+                  In the week leading up to the hackathon, create your own team
+                  or request to join an existing one. Browse the teams on your
+                  location page or the challenge page of your choice.
+                </p>
+              </li>
 
-        </h1>
-      </div> -->
+              <li class="guideline-item">
+                <h4 class="guideline-title">Submit a Project</h4>
+                <p>
+                  Start working on your project once the hackathon begins on
+                  October 1st at 9:00 AM (local time). Before the event ends,
+                  complete your team’s project page with a project description,
+                  data used, and a demonstration.
+                </p>
+              </li>
+            </ol>
+          </div>
+        </div>
+      </div>
     </section>
   </section>
 </template>
 
-<script>
-export default {
-  name: "CompetitionPage",
-};
-</script>
-
 <style scoped>
-@import url("../assets/css/style.css");
+.main {
```

Comment:
The classname `.main` doesn't convey what it is for. Let's use something like `.page-container`